README.md

```diff
@@ -33,10 +33,10 @@ All recommendations here should be be used as a starting point. Further benchmar
 
 For a simple deployment the following configuration works well:
 * Network: Indexer, Conduit, Algod and PostgreSQL should all be on the same network.
-* Indexer: 2 CPU and 8 GB of ram.
-* Conduit: 2 CPU and 8 GB of ram.
+* Indexer: 4 CPU and 8 GB of ram.
+* Conduit: 4 CPU and 8 GB of ram.
 * Database: When hosted on AWS a `db.r5.xlarge` instance works well.
-* Storage: 20 GiB
+* Storage: 20 GiB, 3000 IOPS minimum.
```

Comment:
I might put together Indexer, Conduit, and Storage into one requirement, like `Indexer/Conduit: 4 CPU and 8 GB of RAM, 20 GB storage (3000 IOPS minimum)`

Reply:
Are we recommending to deploy all 3 together, or Conduit/Algod, then Indexer on it's own? I made some adjustments to the Conduit PR assuming Algod+Conduit, but should decide how that impacts this one.

Reply:
For security, they might want to put Indexer on a separate machine -- its the only thing that needs to be exposed to clients. For performance, it can live alongside Conduit/Algod unless read volume is so high that it needs to be elsewhere.

Reply:
I can get behind that approach Will